docs/root/api-v2/output/output.rst

```diff
@@ -0,0 +1,9 @@
+Envoy outputs
```

Comment:
I think I prefer `data`, since we might have some of these as both input and output (e.g. if we want to inject via TAP).

Reply:
OK will switch to data.